### PR TITLE
FAT-211: Implement base test for mod-search

### PR DESCRIPTION
**Pull request by bohdan-suprun**
* Add a data set up feature;
* Add a data destroy feature;
* implement basic search by title.

### CODE REVIEW

**Review by pfilippovEPAM — APPROVED** [review]
Looks good to me.

File: mod-search/src/main/resources/karate-config.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  var env = karate.env;
+
+  // specify runId property for tenant postfix to avoid close connection issues
+  // once we run tests again
+  var runId = karate.properties['runId'];
+
+  var config = {
+    baseUrl: 'https://falcon-okapi.ci.folio.org',
+    admin: {tenant: 'diku', name: 'diku_admin', password: 'admin'},
```
Comment (hjiebsco) [inline mod-search/src/main/resources/karate-config.js:13]: Hardcoded password?

**Review by hjiebsco — COMMENTED** [review]
Is there a way to share `karate-config.js` with other modules? Also, should diku password be embedded?

File: mod-search/src/main/resources/karate-config.js (hunk 1 above)
Comment (bohdan-suprun) [reply mod-search/src/main/resources/karate-config.js:13]: Yes, all modules has hardcoded passwords. No way to inject variables exists now.

Comment (bohdan-suprun) [reply mod-search/src/main/resources/karate-config.js:13]: > Is there a way to share karate-config.js with other modules?

The current approach is that each module is isolated and defines it's own `karate-config.js`

**Review by hjiebsco — COMMENTED** [review]
I am not a fan of hardcoding password and checking into to GitHub. I will defer this to others.

**Comment by bohdan-suprun** [comment]
@hjiebsco, I have created https://issues.folio.org/browse/FOLIO-3178 - to adjust jenkins job and https://issues.folio.org/browse/FAT-916 to apply the properties after the job is updated. 

I'm going to merge this PR.

**Pull request merged**